db/migrate/20181204201916_create_organization_permissions.rb

```diff
@@ -0,0 +1,10 @@
+class CreateOrganizationPermissions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :organization_permissions do |t|
+      t.uuid :organization_id
+      t.boolean :uphold_wallet
+      t.boolean :offline_reporting
+      t.boolean :referral_codes
+    end
```

Comment:
Would you add an index to include `organization_id`?

Reply:
Also, would you set default permissions to `false`?

Reply:
sure thing! 
https://github.com/brave-intl/publishers/pull/1417/commits/c18483e3b2a1235d313ab4596da663d713fe7290